CHANGELOG.md

```diff
@@ -77,6 +77,14 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * New `git.sign-on-push` config option to automatically sign commits which are being
   pushed to a Git remote.
 
+* Publish `config-schema.json` in the documentation site. Users can now reference
```

Comment:
I'd probably say:

> The Jujutsu documentation site now publishes a schema for the official configuration file, which can be integrated into your editor for autocomplete, inline errors, and more. Please [see the documentation](...) for more on this.

i.e. we normally would link to the docs elsewhere, rather than including the example inline.

Reply:
Sounds good, thank you!! I wasn't quite sure about how specific to be here. Will update shortly.

Reply:
Done! Thanks for your help!